typescript/phoenix-sdk/examples/placeLimitOrder.ts

```diff
@@ -41,21 +48,21 @@ export async function placeLimitOrderExample() {
   // - Create associated token accounts for base and quote tokens, if needed
   // - Claim a maker seat on the market, if needed
   const setupNewMakerIxs = await PhoenixSdk.getMakerSetupInstructionsForMarket(
-    phoenixClient,
-    marketAddress,
-    marketData,
+    connection,
+    market,
     traderKeypair.publicKey
   );
   const setupTx = new Transaction().add(...setupNewMakerIxs);
 
-  const setupTxId = await phoenixClient.connection.sendTransaction(
+  const setupTxId = await sendAndConfirmTransaction(
+    connection,
     setupTx,
     [traderKeypair],
     {
       skipPreflight: true,
+      commitment: "confirmed",
     }
   );
-  await phoenixClient.connection.confirmTransaction(setupTxId, "confirmed");
   console.log(
     `Setup Tx Link: https://beta.solscan.io/tx/${setupTxId}?cluster=devnet`
   );
```

Comment:
can you  make this fix on everything in the `examples` directory?

Reply:
I thought I did?

Reply:
ty, seems fine, hadn't gotten that far yet :)